dev/docker-compose.yml.example

```
@@ -30,7 +30,10 @@ services:
 
   db:
     restart: always
-    image: docker.io/postgres:12.2-alpine
+    image: postgres:15.3-alpine
+		build:
+      context: ../
+      dockerfile: Dockerfile.db
```

Thoughs ont moving the `Dockerfile.db` to the `dev/` folder and updating this line to this?

```suggestion
      dockerfile: ./Dockerfile.db
```

---

The db file is needed for the docker compose at the root of the project to work. I've updated the path here so it references the file correctly.